### ISSUE TITLE
Over exposed lighting in 1.10.6 (not 1.10.7)

### ISSUE THREAD

**Issue by QSpukee** [ButteryStancakes/ButteryFixes#18]
Most recent update (~1.10.7~ 1.10.6) caused light to become overexposed - I toggled the lighting fix logic from true -> false -> true there wasn't a difference. There are a few packs I have with this mod, every one was affected by it including the packs that do not have many lighting mods to them. If requested I can add screenshots I didn't take them earlier.

**Comment by ButteryStancakes** [comment]
Are you using SpaceSunShine? Or any other mods that affect the appearance of the space view outside of the ship?

There was an incompatibility with CelestialTint in v1.10.6 but that is what v1.10.7 was released to fix.

**Comment by QSpukee** [comment]
My apologies it was  v1.10.6, the version released on Oct 31st when I was having the issue with light exposure then reported the bug later. I took the 1.10.7 version number not realizing there was already a fix. I do use CelestialTint and tried the mod pack a few hours ago and it works just fine! Thanks for cheking!


I updated the title and the comment to reflect the version that was causing the issue though. Closing as fixed. 

Havd a good day! ^w ^
